fractal_tasks_core/ngff/specs.py

```diff
@@ -150,13 +161,19 @@ class Multiscale(BaseModel):
             ]
         ]
     ] = None
+    _check_unique = field_validator("axes")(unique_items_validator)
 
-    @validator("coordinateTransformations", always=True)
-    def _no_global_coordinateTransformations(cls, v):
+    @field_validator("coordinateTransformations", mode="after")
+    @classmethod
+    def _no_global_coordinateTransformations(
+        cls, v: Optional[list]
+    ) -> Optional[list]:
         """
         Fail if Multiscale has a (global) coordinateTransformations attribute.
         """
-        if v is not None:
+        if v is None:
```

Comment:
It looks like this `if v is None` branch is unreachable. Maybe this is due to the `mode="after"` option or to some other Pydantic option? TBD

Reply:
Now tested with 09b76b212681ad46ea8c2efb3480f2a12e40c658.
The reason why it seemed unreachable is the difference between `None` and unset. Now I included both in the test.